cpp/benchmarks/CMakeLists.txt

```diff
@@ -15,15 +15,24 @@
 #=============================================================================
 cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
```

Comment:
Since you're only calling this via `add_subdirectory` you should be safe to remove this line